ci-operator/config/openshift/knative-client/openshift-knative-client-release-next__4.10.yaml

```diff
@@ -59,7 +57,11 @@ images:
       - destination_dir: .
         source_path: /go/bin/kn
   to: knative-client
+- dockerfile_path: openshift/ci-operator/test-e2e-image/Dockerfile
```

Comment:
Is this new, or related to the `ubi build image`?

Reply:
We still need some of the tools to be present for e2e tests, see my comment above. https://github.com/openshift/knative-client/commit/7ff2766b97d2e37908a468de9a492fac6dea81ee

Reply:
@dsimansk is the idea to go back to using `openshift/ci-operator/build-image/Dockerfile` once actual branches are migrated to use `FROM src`?

Reply:
I assume you mean `build-image` with `FROM src` would be equal to `test-e2e-image` content-wise.

I was thinking about it, but it might be confusing name, because of CI docs mention "builder image" or "build root" in a context of base image for the CI run. Therefore I'd just keep a different more specific name, e.g. `test-e2e-images` or whatever we feel is suitable, when we switch to imageStream `build_root` definition.

```
build_root:
  image_stream_tag:
    name: release
    namespace: openshift
    tag: golang-1.18
```